crates/interpreter/src/valid.rs

```diff
@@ -414,6 +414,8 @@ struct Expr<'a, 'm> {
 struct SideTableBranch<'m> {
     parser: &'m [u8],
     side_table: usize,
+    val_cnt: u32,
+    pop_cnt: u32,
```

Comment:
This should probably be something like
```suggestion
    stack: usize,
```
And be initialized by `stack.len()` (like `side_table` is initialized with `side_table.len()`).

We probably need a function to compute the stack length (going through all labels). We don't need to compute across functions because we can only jump within a function. We could try to memoize the sum of previous label in each label.

Reply:
Does the `stack` similar to the [`Label::values_cnt` in exec.rs](https://github.com/zhouwfang/wasefire/blob/15b120fa5426d34f27e1025c4db31f521b37c10c/crates/interpreter/src/exec.rs#L1433)?

Reply:
Yes, that's the value stack. There's one difference though, is that in `exec.rs` it's the value stack of the thread (i.e. function and label stack). Here we only need the value stack of the function (i.e. label stack).